components/pipeline-service/staging/base/kustomization.yaml

```diff
@@ -20,4 +20,9 @@ patches:
     target:
       kind: ConfigMap
       name: pipelines-as-code
+      namespace: openshift-pipelines
+  - path: pac-app-secret.yaml
+    target:
+      kind: ExternalSecret
+      name: pipelines-as-code-secret
       namespace: pipelines-as-code
```

Comment:
@Roming22 we still have this one that should be replaced with openshift-pipelines probably

Reply:
Ah no that's irrelevant, it is actually the patch you made to update the ns.
I though it would be also better to fix the ns directly here https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/base/external-secrets/pipelines-as-code/kustomization.yaml. However we can do that only when prod is also updated this way we can have the right ns directly in the base files

Reply:
@Mo3m3n I don't believe so. I cannot change the namespace directly in `components/pipeline-service/base/external-secrets/pipelines-as-code/pipelines-as-code-secret.yaml` as it would impact production as well.

Therefore I need to apply a patch to the resource that is initially in the old namespace. It is in the right namespace after the patch is applied.
```
kind: ExternalSecret
metadata:
  annotations:
    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
    argocd.argoproj.io/sync-wave: "-1"
  name: pipelines-as-code-secret
  namespace: openshift-pipelines
spec:
  dataFrom:
  - extract:
      key: staging/pipeline-service/github-app
```

Reply:
yes that's why I have concluded with:
> However we can do that only when prod is also updated this way we can have the right ns directly in the base files